src/views/home/UserInput.js

```diff
@@ -76,14 +76,36 @@ export default class UserInput extends Component {
     e.preventDefault();
     this.setState(prevState => {
       let ticket = { ...prevState.ticket };
-      ticket.screenshots = ticket.screenshots.filter((t, index) => i !== index);
+      ticket.screenshots = ticket.screenshots.filter((s, index) => i !== index);
       return { ticket };
     });
   };
 
+  cleanScreenshots = ticket => {
+    const isValidScreenshot = s => {
+      return typeof s.height === 'number' && typeof s.width === 'number';
+    };
+
+    let newTicket = { ...ticket };
+    // filter out screenshots with no width or height
+    newTicket.screenshots = newTicket.screenshots.filter(s =>
```

Comment:
`filter(isValidScreenshot)` is sufficient

Reply:
yeah just noticed that and updated